editor/editor_fonts.cpp

```diff
@@ -226,6 +227,23 @@ void editor_register_fonts(Ref<Theme> p_theme) {
 	default_fc->set_spacing(TextServer::SPACING_TOP, -EDSCALE);
 	default_fc->set_spacing(TextServer::SPACING_BOTTOM, -EDSCALE);
 
+	Ref<FontVariation> default_theme_fc;
+	default_theme_fc.instantiate();
+	if (custom_font_path.length() > 0 && dir->file_exists(custom_font_path)) {
```

Comment:
```suggestion
	if (!custom_font_path.is_empty() && dir->file_exists(custom_font_path)) {
```

Reply:
doesn't `is_empty()` check for `length() >0` ?

Reply:
Yes, we just prefer it by convention over checking size.

Reply:
ohk